_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -20,15 +20,28 @@ acknowledgements:
 
 The rule applies to any [keyboard event][]: 
  - where the attribute `key` is a [printable character][] key; and
- - the attribute `isComposing` is set to `false`; or the [keyboard event][] is part of an [composition event][] whose attribute `data` includes at least one [non-printable character][] key; and
+ - the method `getModifierState` returns `false`; and
  - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation
+## Expectation 1
 
 For each test target:
- - an [instrument][] to prevent the outcome of the [keyboard event][] is available; or
- - if the [event target][] does not have [focus], the outcome of the event is prevented; or
- - an [instrument][] is available to force the [keyboard event][] to be part of an [composition event][] whose attribute `data` must include at least one [non-printable character][] key.
+ - an [instrument][] to prevent any result of the [keyboard event][] is available; or
+ - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
```

Comment:
There probably needs some assumption as to which results are considered. See this discussion https://github.com/act-rules/act-rules.github.io/pull/880#discussion_r424982737 but that rule does have an assumption to tackle the problem.